config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryanalyticshublistings.bigqueryanalyticshub.cnrm.cloud.google.com.yaml

```diff
@@ -200,27 +194,29 @@ spec:
                       type: string
                   type: object
                 type: array
-              name:
-                description: The resource name of the listing. e.g. "projects/myproject/locations/US/dataExchanges/123/listings/456".
+              externalRef:
+                description: A unique specifier for the BigQueryAnalyticsHubDataExchangeListing
+                  resource in GCP.
                 type: string
               observedGeneration:
                 description: ObservedGeneration is the generation of the resource
                   that was most recently observed by the Config Connector controller.
                   If this is equal to metadata.generation, then that means that the
                   current reported status reflects the most recent desired state of
                   the resource.
+                format: int64
                 type: integer
+              observedState:
+                description: ObservedState is the state of the resource as most recently
+                  observed in GCP.
+                properties:
+                  state:
+                    description: Output only. Current state of the listing.
+                    type: string
+                type: object
             type: object
-        required:
-        - spec
```

Comment:
Nit: It can be easier to add the `// +required` to make the review more deterministic!